config.js

```diff
@@ -0,0 +1,75 @@
+// NOTE: Neither of these files are committed and for good reason.
```

Comment:
I'm planning to make which config.js you load come from the command line, just like it used to. If that's the case, you can have a tesster.js which loads the appropriate versions of these. 

Alternatively, it doesn't matter. I never want the live token/keys on my dev environment, and I don't want the testing ones in the same directory structure on the live environment.  So I'm not sure how important it is?  

Even having said that, you're probably right. I should probably make it easier to switch between the two.